keycloak/keycloak-admin-ui
# Add button on authentication flows

## Motivation
This adds the add button on the authetication flows [marvel](https://marvelapp.com/prototype/bh91013/screen/75676609)

## Brief Description

Adding an execution or condition works, but subflow only shows a model that doesn't work, as it needs some clarification
on how to do that without type and provider.

## Verification Steps

1. Go to authentication copy one of the existing flows
1. Then go to the details of that flow
1. Now you'll see a `+` icon on the sub flows
1. use the "Add step" and "Add condition" dropdown items to add excutions to the sub flow

## Checklist:


## Additional Notes
